src/core/Akka.Remote.TestKit/Internals/Reference.conf

```diff
@@ -43,7 +43,7 @@ akka {
         pool-size-factor = 1.0
 
         # Max number of threads to cap factor-based number to
-        pool-size-max = 2
+        pool-size-max = 1
```

Comment:
The internal Helios connection used in the MNTR for things like the shared synchronization barriers now run on one thread - probably doesn't impact much since messaging guarantees aren't as important here but I thought it best to be consistent.